src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -179,6 +184,15 @@ public void parse_repeatedNonTagValue_failure() {
                 Messages.getErrorMessageForDuplicatePrefixes(PREFIX_DOB));
     }
 
+    @Test
+    public void parse_optionalFieldsMissing_success() {
+        // zero tags
+        Person expectedPerson = new PersonBuilder(AMY).withTags().withMedCons().build();
+        assertParseSuccess(parser, NAME_DESC_AMY + PHONE_DESC_AMY + EMAIL_DESC_AMY + ADDRESS_DESC_AMY
+                + NRIC_DESC_AMY + DOB_DESC_AMY + GENDER_DESC_AMY + PRIORITY_DESC_AMY,
+                new AddCommand(expectedPerson));
+    }
+
```

Comment:
I don' t think there is a need for this test case anymore since `Tag` is no longer part of `add` command